js/app.js

```diff
@@ -199,20 +197,16 @@ var data = {
       'rgb(229, 226, 60)',
       'rgb(202, 239, 15)'
     ],
-    borderColor: [
-    ],
-    borderWidth: [
-    ]
+    borderColor: [],
+    borderWidth: []
```

Comment:
If you are not setting these values, they do not need to be included in the `data`.